bin/gce-align.sh

```diff
@@ -21,106 +24,122 @@ shift
 OUTPUT_DIR=$1
 shift
 INPUT_FILES=$@
-
-set -o pipefail
+INPUT_FILES_COUNT=$#
 
 case "$PAIRED" in
   0) BWA_PAIRED_OPT="" ;;
   1) BWA_PAIRED_OPT="-p" ;;
 esac
 
-MACHINE_NAME="align-${SAMPLE_ID}-${RG_IDENT}"
-MACHINE_TYPE_OPTS="--custom-cpu 32 --custom-memory 64GiB"
-
-gcloud compute instances create --scopes storage-full --image ubuntu-1404-alignment $MACHINE_TYPE_OPTS --boot-disk-size 300 --preemptible $MACHINE_NAME
+ZONES[0]="us-central1-b"
+ZONES[1]="us-central1-c"
+ZONES[2]="us-central1-f"
+MACHINE_ZONE=${ZONES[$[ $RANDOM % 3 ]]}
 
-EXIT_STATUS=$?
-if [[ $EXIT_STATUS == 0 ]]
-then
-  # Give ssh daemon some time to get running.
-  sleep 30s
+MACHINE_NAME=$(echo "align-${SAMPLE_ID}-${RG_IDENT}" | tr "[:upper:]" "[:lower:]" | sed "s/[^a-z0-9]/-/g" | head -c62)
```

Comment:
This isn't in gce-align.sh currently and in cases like this I'd prefer that the script just exit if the args don't meet the criteria, but whatever.